cardano-explorer-db/src/Explorer/DB/Types.hs

```diff
@@ -10,14 +11,28 @@ module Explorer.DB.Types
   ) where
 
 
-import           Data.Fixed (Micro)
+import           Data.Aeson.Encoding       (unsafeToEncoding)
+import           Data.Aeson.Types          (ToJSON (toEncoding, toJSON))
+import qualified Data.ByteString.Builder   as BS (string8)
+import           Data.Fixed (Micro, showFixed)
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import           Data.Word (Word64)
+import           GHC.Generics              (Generic)
 
 newtype Ada = Ada
   { unAda :: Micro
-  } deriving (Eq, Ord, Show)
+  } deriving (Eq, Ord, Generic)
+
+instance ToJSON Ada where
+    -- https://github.com/bos/aeson/issues/227#issuecomment-245400284
+    toEncoding (Ada ada) =
+        unsafeToEncoding $   -- convert ByteString to Aeson's Encoding
+        BS.string8 $         -- convert String to ByteString using Latin1 encoding
+        showFixed True ada   -- convert Micro to String chopping off trailing zeros
```

Comment:
Pretty sure that can just be done as:
```
    Aeson.String $ Text.pack (showFixed True ada)
```
Where `String` is the `Text -> Value` constructor.